clang/lib/CIR/Dialect/IR/CIRDialect.cpp

```diff
@@ -63,9 +63,9 @@ struct CIROpAsmDialectInterface : public OpAsmDialectInterface {
     if (auto recordType = dyn_cast<cir::RecordType>(type)) {
       StringAttr nameAttr = recordType.getName();
       if (!nameAttr)
-        os << "ty_anon_" << recordType.getKindAsStr();
+        os << "rec_anon_" << recordType.getKindAsStr();
```

Comment:
not to do in this patch, but we discussed making this an 'assert'  at one point, right?

Reply:
Yeah. I was going to do that separately.